src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -48,6 +48,15 @@
                 "JobPollingFrequency": "00:00:10",
                 "MaximumNumberOfResourcesPerQuery": 100,
                 "NumberOfPagesPerCommit": 10
+            },
+            "Reindexing": {
+                "Enabled": false,
+                "ConsecutiveFailuresThreshold": 5,
+                "DefaultMaximumThreadsPerReindexJob": 1,
+                "MaximumNumberOfResourcesPerQuery": 100,
+                "JobHeartbeatTimeoutThreshold": "00:10:00",
+                "JobPollingFrequency": "00:00:10",
+                "QueryDelayIntervalInMilliseconds": 500
```

Comment:
`ReindexJobConfiguration` does not have `QueryDelayIntervalInMilliseconds`

Reply:
Added, thanks for the catch.